stock_picking_package_preparation_line/models/stock_picking_package_preparation_line.py

```diff
@@ -206,7 +206,8 @@ def action_put_in_pack(self):
             for line in package.line_ids:
                 # ----- If line has 'move_id' this means we don't need to
                 #       recreate picking and move again
-                if line.product_id and not line.move_id:
+                if (line.product_id and line.product_id.type != 'service'
```

Comment:
OK, I won't block because it's harmless, but an stock.move can also only be associated to a non service product AFAIK